include/emp/prefab/ConfigPanel.hpp

```diff
@@ -106,33 +102,12 @@ namespace prefab {
       }
 
       inline static std::set<std::string> numeric_types = {"int", "double", "float", "uint32_t", "uint64_t", "size_t"};
-      Config & config;
-      std::set<std::string> excluded_settings;
-      std::set<std::string> excluded_groups;
-      std::map<std::string, web::Div> input_divs;
-
-      std::function<std::string(std::string val)> format_label_fun = [](std::string name) {
-        emp::vector<std::string> sliced = slice(name, '_');
-        return to_titlecase(join(sliced, " "));
+      // Helper function to get prety names from config values
+      inline static std::function<std::string(std::string val)> format_label = [](std::string name) {
+        return to_titlecase(join(slice(name, '_'), " "));
       };
 
-      /**
-       * For ints and doubles, there are three inputs:
-       * a number input and two slider inputs.
-       * SyncForm updates two inputs with the new value in the third input.
-       *
-       * @param val new value to be update the inputs with
-       * @param input1 ID of one input that needs its value updated
-       * @param input2 ID of the second input that needs its value updated
-       */
-      void SyncForm(const std::string val, const std::string input1, const std::string input2) {
-          emp::web::Input div1(this->Find(input1));
-          div1.Value(val);
-          emp::web::Input div2(this->Find(input2));
-          div2.Value(val);
-          div1.Redraw();
-          div2.Redraw();
-      }
+      bool init_open;
```

Comment:
Does this need to be class data? Why does this need to be stored past construction?

Reply:
Nope, I'll remove it